dashboards/service-level.jsonnet

```diff
@@ -35,10 +35,11 @@ stdlib.dashboard(
 )
 
 .addTemplates([
-    stdlib.variable('service', 'Service', 'label_values(slo_requests{pipeline="stable"}, service)'),
-    stdlib.variable('customer', 'Customer', 'label_values(slo_requests{pipeline="stable"}, customer)'),
-    stdlib.variable('installation', 'Installation', 'label_values(slo_requests{pipeline="stable", customer=~"$customer"}, installation)'),
-    stdlib.variable('cluster_id', 'Cluster ID', 'label_values(slo_requests{pipeline="stable", installation=~"$installation", customer=~"$customer"}, cluster_id)'),
+    stdlib.variable('pipeline', 'Pipeline', 'label_values(slo_requests, pipeline)', current='stable'),
```

Comment:
This is the only 'new' part, the new variable to allow for selecting the pipeline type. Rest is updating queries to use new pipeline variable